### PR TITLE
Backport #25786

### LINKED ISSUE #39: Inspection with `?` raises an ImportError
```
sage: ZZ?
ImportError: No module named conf
```

ArchLinux has the same issue btw.

### PR DESCRIPTION
**Pull request by saraedum**
there has been no movement on that ticket but I think it's ready, so we should backport it.

Fixes #39.

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender.

**Comment by saraedum** [comment]
@isuruf:
I get
```
+ set -e
+ conda build ./recipe -m ./.ci_support/osx_python3.6.yaml
+ '[' 4 -lt 1 ']'
+ local cmd=build
+ shift
+ case "$cmd" in
+ /Users/distiller/miniconda3/bin/conda build ./recipe -m ./.ci_support/osx_python3.6.yaml
Traceback (most recent call last):
  File "/Users/distiller/miniconda3/bin/conda-build", line 7, in <module>
    from conda_build.cli.main_build import main
  File "/Users/distiller/miniconda3/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 18, in <module>
    import conda_build.api as api
  File "/Users/distiller/miniconda3/lib/python3.6/site-packages/conda_build/api.py", line 22, in <module>
    from conda_build.config import Config, get_or_merge_config, DEFAULT_PREFIX_LENGTH as _prefix_length
  File "/Users/distiller/miniconda3/lib/python3.6/site-packages/conda_build/config.py", line 17, in <module>
    from .variants import get_default_variant
  File "/Users/distiller/miniconda3/lib/python3.6/site-packages/conda_build/variants.py", line 15, in <module>
    from conda_build.utils import ensure_list, trim_empty_keys, get_logger
  File "/Users/distiller/miniconda3/lib/python3.6/site-packages/conda_build/utils.py", line 10, in <module>
    import libarchive
  File "/Users/distiller/miniconda3/lib/python3.6/site-packages/libarchive/__init__.py", line 1, in <module>
    from .entry import ArchiveEntry
  File "/Users/distiller/miniconda3/lib/python3.6/site-packages/libarchive/entry.py", line 6, in <module>
    from . import ffi
  File "/Users/distiller/miniconda3/lib/python3.6/site-packages/libarchive/ffi.py", line 27, in <module>
    libarchive = ctypes.cdll.LoadLibrary(libarchive_path)
  File "/Users/distiller/miniconda3/lib/python3.6/ctypes/__init__.py", line 426, in LoadLibrary
    return self._dlltype(name)
  File "/Users/distiller/miniconda3/lib/python3.6/ctypes/__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(/Users/distiller/miniconda3/lib/libarchive, 6): image not found
```

Is building for OSX on CircleCI broken?

**Comment by isuruf** [comment]
Can you open an issue in python-libarchive-c-feedstock ?

**Comment by saraedum** [comment]
Great. That fixed it.

File: recipe/patches/25786.patch [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+     doctreedir = os.path.join(srcdir, 'doctrees')
+     confoverrides = {'html_context': {}, 'master_doc': 'docstring'}
+ 
+--- a/src/sage_setup/docbuild/build_options.py
++++ b/src/sage_setup/docbuild/build_options.py
+@@ -9,7 +9,6 @@
+ LANGUAGES = [d for d in os.listdir(SAGE_DOC_SRC) if re.match('^[a-z][a-z]$', d)]
+ SPHINXOPTS = ""
+ PAPER = ""
+-OMIT = ["introspect"]  # docs/dirs to omit when listing and building 'all'
```
Comment (isuruf) [inline recipe/patches/25786.patch]: @saraedum, this is used in `src/sage_setup/docbuild/__init__.py` and `src/sage/misc/sagedoc.py`

**Pull request merged**